### PR TITLE
relationships page updates

### PR DESCRIPTION
**Pull request by kristinmerbach**
# PR Checklist

Please check if your PR fulfills the following requirements
- [ ] The title follows our [guidelines](https://github.com/ideacrew/enroll/blob/trunk/CONTRIBUTING.md#commit)
- [ ] Tests for the changes have been added (for bugfixes/features), they use let helpers and before blocks
- [ ] For all UI changes, there is cucumber coverage
- [ ] Any endpoint touched in the PR has an appropriate Pundit policy. For open endpoints, reasoning is documented in PR and code
- [ ] Any endpoint modified in the PR only responds to the expected MIME types.
- [ ] For all scripts or rake tasks, how to run it is documented on both the PR and in the code
- [ ] There are no inline styles added
- [ ] There are no inline javascript added
- [ ] There is no hard coded text added/updated in helpers/views/Javascript. New/updated translation strings do not include markup/styles, unless there is supporting documentation
- [ ] Code does not use .html_safe
- [ ] All images added/updated have alt text
- [ ] Doesn’t bypass rubocop rules in any way

# PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature (requires Feature flag)
- [ ] Data fix or migration (inert code, no impact until run)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Dependency updates (e.g., add a new gem or update to a version)

# What is the ticket # detailing the issue?

Ticket: https://www.pivotaltracker.com/story/show/188013340
https://www.pivotaltracker.com/story/show/188014923
https://www.pivotaltracker.com/story/show/188014869
https://www.pivotaltracker.com/story/show/188013380

# A brief description of the changes

Current behavior:

New behavior:

# Feature Flag

For all new feature development, a feature flag is required to control the exposure of the feature to our end users. A feature flag needs a corresponding environment variable to initialize the state of the flag. Please share the name of the environment variable below that would enable/disable the feature and which client(s) it applies to.

Variable name:

- [ ] DC
- [ ] ME

# Additional Context
Include any additional context that may be relevant to the peer review process.

# AppScan CodeSweep Failure
In the event of a failed check on the AppScan CodeSweep step of our GitHub Actions workflow, please review the False Positive protocol outlined here: appscan_codesweep/CODESWEEP_FALSE_POSITIVES_README.MD

Add all required notes to this section if the failure is a suspected false positive.

### CODE REVIEW

**Review by TristanB17 — APPROVED** [review]
lgtm!

File: components/financial_assistance/db/seedfiles/translations/en/me/financial_assistance.rb [hunk 1]
```diff
@@ -342,7 +342,11 @@
   "en.faa.review.other_questions" => "Other Questions",
   "en.faa.review.coverage.caretaker" => "Which member(s) of the household is this person the caretaker for?",
   "en.faa.review.your_household" => "Your Household",
+  "en.faa.review.your_household.relationship_lanel" => "Relationship",
+  "en.faa.review.your_household.is" => "is",
+  "en.faa.review.your_household.of" => "of",
   "en.faa.review.your_household.relationship" => "%{related_name} is %{relationship} of",
```
Comment (charlienparker) [inline components/financial_assistance/db/seedfiles/translations/en/me/financial_assistance.rb:348]: can we remove the new "is" and "of" strings in favor of the `en.faa.review.your_household.relationship`?

**Review by charlienparker — APPROVED** [review]
approved with small comment - seems a translation key is incorrect

File: components/financial_assistance/app/views/financial_assistance/relationships/_form.html.erb [hunk 2]
```diff
@@ -1,29 +1,49 @@
...
-        <h3>
-          <label class="required" for="relationships-<%= er[:relative] %>"><%= l10n("relationship_to_primary") %></label>
-        </h3>
-        <%= f.select :relationships, er[:relation].present? ? options_for_select(humanize_relationships, er[:relation]) : options_for_select(humanize_relationships), {:prompt => 'Select Option', required: true }, { "class": "focus input-field #{'invalid-relation' if er[:relation] == 'none'}", "data-applicant": er[:applicant], "data-relative": er[:relative], id: "relationships-#{er[:relative]}" } %>
+  <%= form_tag application_relationships_path(@application), remote: form_remote do %>
+    <% @all_relationships.each do |er| %>
+      <div class="d-flex justify-items-between my-4 align-items-center" id="<%= "#{er[:applicant] }_#{er[:relative]}" %>">
+        <span><%= @application.applicants.find(er[:applicant]).first_name %> <%= l10n("faa.review.your_household.is") %></span>
+        <div class="mx-3">
+          <label for='<%= "#{er[:applicant] }_#{er[:relative]}_relationship" %>' class="sr-only"><%= l10n("faa.review.your_household.relationship_lanel") %></label>
```
Comment (charlienparker) [inline components/financial_assistance/app/views/financial_assistance/relationships/_form.html.erb:25]: is `relationship_lanel` correct?

Comment (kristinmerbach) [reply components/financial_assistance/app/views/financial_assistance/relationships/_form.html.erb:25]: I'll update the key, it is at least wrong on both the view and the translation file

**Review by charlienparker — APPROVED** [review]

**Pull request merged**